setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = lto-cli
-version = 0.0.10
+version = 0.0.11
```

Comment:
Do you update the version on every PR?

Reply:
same on the other PR, you have to, not my choice